tools/bazel_rules/jest/jest-runner.js

```diff
@@ -1,17 +1,24 @@
 const { runCLI } = require('jest');
-const { resolve, join, dirname, parse } = require('path');
+const { resolve, join, dirname, parse, sep } = require('path');
 const { readFileSync, writeFileSync } = require('fs');
 const { argv } = require('yargs');
 
+// Path of the test.sh binary that gets created from the nodejs_test
+const { TEST_BINARY } = process.env;
+
 /**
  * Updates the jest config json with additional properties
  * @param {string} configPath Path to the jest config json
  * @param {string} suiteName The name of the test suite
  * @param {string} suitePath The path of the package
  */
-function updateJestConfig(configPath, suiteName, suitePath) {
+function updateJestConfig(suiteName, config = undefined) {
+  const suitePath = dirname(config || TEST_BINARY);
```

Comment:
Wouldn't the null-coalescing operator _??_ be more suitable here?

Reply:
Agree 👍